custom/icds/tasks/__init__.py

```diff
@@ -25,7 +25,7 @@ def delete_old_images():
 @task
 def delete_old_images_on_db(db_name, cutoff):
     if isinstance(cutoff, str):
-        cutoff = parse_date(cutoff)
+        cutoff = parse_date(cutoff).replace(tzinfo=None)
```

Comment:
When will cutoff ever be a string? I took a peek at the code and it seems to only ever have a value that comes from `datetime.utcnow()` or `datetime.utcnow() - cutoff`, neither of which should ever be a string.

Just saying that unless the problem is deeper/weirder than this.

If cutoff is supposed to be a datetime string with a timezone, then `UserTime(cutoff).server_time().done()` would work to make sure it's shifted into UTC first and then turned into an offset-naive datetime. If it's always going to be a datetime string ending in 'Z', then this is equivalent to that.

Reply:
it's a string because Celery serializes the datetime to JSON but then doesn't deserialize it. There's no need for timezone conversion. Going to update it to this though:
```suggestion
        cutoff = parse_date(cutoff, default_timezone=None)
```